smoke-test.sh

```diff
@@ -212,17 +212,25 @@ function doCommand() {
                 "${dotnetCmd}" $newArgs --no-restore >> "$logFile" 2>&1
             fi
         elif [[ "$1" == "run" && "$proj" =~ ^(web|mvc|webapi|razor)$ ]]; then
+            # A separate log file that we will over-write all the time.
+            exitLogFile="$testingDir/exitLogFile"
+            echo > $exitLogFile
             if [ "$projectOutput" == "true" ]; then
                 "${dotnetCmd}" $1 &
```

Comment:
This is strange... it doesn't write the output to any file, so if `$projectOutputt` is `true`, the timeout logic will still break?

Reply:
Interesting... and it looks like CI always ends up setting `--projectOutput`, so maybe this never actually worked in CI. I did *notice* the issue while running locally without `--projectOutput`, so that seems possible. Other cases use `tee` to output to both places, not sure why this one isn't. @dseefeld @crummel ?

Reply:
I vaguely remember `tee` and sending SIGINT not interacting well in my 1.x prototype of this script.  I think it might have been copied wholesale and we just never changed it for 2.x or when we changed the background monitoring to fix it.